README.md

```diff
@@ -114,6 +114,43 @@ across the entire platform.
 
 Amazon XRAY also has a unique tracing ID that is propagated across the requests and can be tracked as well.
 
+## Customize request tracing
+
+You can customize the data that is tracked for each request by adding a global request mixin. 
```

Comment:
Tiny nit: `global` here made me think that the mixin applied like a singleton to all constructed `pino` instances. I think L121-122 explain the actual semantics well–maybe `per-request mixin` would work here instead?